test/contrib/rack/helpers.rb

```diff
@@ -1,9 +1,12 @@
 require 'helper'
+require 'net/http'
 require 'ddtrace'
 require 'ddtrace/contrib/rack/middlewares'
 
 require 'rack/test'
 
+Datadog::Monkey.patch_module(:http)
```

Comment:
You're actually right. The only thing is... the way the test framework (and our setup works) I could technically write another file and requiring it here only but anyway minitest is going to load it whatsoever and monkey patch http anyway. The only reasonable way I see to fix it is to write another test, elsewhere, in another directory, but then I'd copy/paste quite a bit of code. This is why I did it here. But not so proud of it.